docs/tools/tools_intro.md

```diff
@@ -44,3 +45,30 @@ For more information, please see the [Online Compiler page](developing-mbed-onli
 
 You can export your project from any of our tools to third party tools. For instructions, as well as tool-specific information, see [the Exporting to third party toolchains page](exporting.html).
 
+<div style="background-color:#F3F3F3; text-align:left; vertical-align: middle; padding:15px 30px;"> **Note:** We encourage you to switch to Arm Compiler 6 soon because we will deprecate Arm Compiler 5 support in the future. However, if you need to update to Mbed OS 5.12 but still require compiling with Arm Compiler 5 until you are in possession of Arm Compiler 6, we provide methods to override the Arm toolchain version. If you do this, your target may not be able to compile with Arm Compiler 5, or you may see undefined behaviors.
+
+To force Arm Compiler 5, you can use the following options:
+
+- Create or update your `mbed_app.json` with:
+
+```
+{
+  "target_overrides": {
+      "*": {
+          "target.supported_toolchains": ["ARMC5", "GCC_ARM", "IAR"]
+      }
+  }
+}
+```
+
+- Modify the `supported_toolchains` entry in targets.json to replace all `ARM`, `ARMC6` entries with `ARMC5`: 
```

Comment:
Shall I create a new PR moving this to porting?

Reply:
Or would config be a better location? We do talk about target.json here: https://os.mbed.com/docs/mbed-os/development/reference/adding-and-configuring-targets.html

Reply:
That's porting documentation. It talks about how to port targets.